tests/wms_test_utils.py

```diff
@@ -0,0 +1,61 @@
+# This file is part of ctrl_bps.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+import logging
+
+from lsst.ctrl.bps.wms_service import BaseWmsService
+
+_LOG = logging.getLogger(__name__)
+
+
+class WmsServiceSuccess(BaseWmsService):
```

Comment:
If the intent is for these to be used in other plugins then they have to be moved to `lsst.ctrl.bps.tests` or something in the main package. Other plugins can't see this code.

Reply:
At the moment I cannot think of why the plugin packages would ever use these specific classes.  These classes only exist because I am faking a plugin for the core bps tests.  I would have put them in the same file as the tests, but because bps takes a string and calls `doImport` I couldn't get it to work until moving them to a separate file.

Reply:
I will keep thinking on this while making the other changes.  @mxk62  Can you think of a case where plugin tests would use these fake plugin classes?  Or maybe it's the other side.  Maybe campaign tooling could use fake plugin classes.

Reply:
The way you get it to work inline is to create the class and then use `lsst.utils.introspection.get_full_type_name` to work out the name of that class -- then python will realize that it has already imported it when doImport is called (unless the doImport is done after forking a subprocess).

Reply:
Postponing location decision to the ticket for writing tests for the other bps drivers (DM-35089).